docs/src/hybrid_search.md

```diff
@@ -0,0 +1,172 @@
+# Hybrid Search
+
+2 common serching techiniques are semantic and keyword-based search. LanceDB supports both of these approaches. In real world applications, it is often useful to combine these two approaches to get the best best results. For example, you may want to search for a document that is semantically similar to a query document, but also contains a specific keyword. This is an example of hybrid search. Hybrid search simply means a searching algorithm that combines multiple search techniques. The most common ones are Semantic and Full-text search but it can be extended to other techniques as well.
+
+## Hybrid search in LanceDB
+You can perform hybrid search in LanceDB by combining the results of semantic and full-text search using the re-ranking algorithm of your choice. You can either choose one of the many re-rankers supported by lancedb or you can write your own custom re-ranker.
+
+```python
+import lancedb
+from lancedb.embeddings import get_registry
+from lancedb.pydanatic import LanceModel, Vector
+
+db = lancedb.connect("~/.lancedb")
+
+# Ingest embedding function in LanceDB table
+embeddings = get_registry().get("openai").create()
+
+class Documents(LanceModel):
+    vector: Vector(embeddings.ndims) = embeddings.VectorField()
+    text: str = embeddings.SourceField()
+
+table = db.create_table("documents", schema=Documents)
+
+data = [
+    { "text": "rebel spaceships striking from a hidden base"},
+    { "text": "have won their first victory against the evil Galactic Empire"},
+    { "text": "during the battle rebel spies managed to steal secret plans"},
+    { "text": "to the Empire's ultimate weapon the Death Star"}
+    ]
+
+# ingest docs with auto-vectorization
+table.add(data)
+
+# hybrid search with default re-ranker
+results = table.search("flower moon", query_type="hybrid").to_pandas()
+```
+
+By default, LanceDB uses `LinearCombinationReranker(weights=0.7)` to combine & Rerank the results of semantic and full-text search. You can customize the hyperparameters, use a different re-ranker or write your own custom re-ranker. Here's how you can use any of the available re-rankers:
```

Comment:
```suggestion
By default, LanceDB uses `LinearCombinationReranker(weights=0.7)` to combine and rerank the results of semantic and full-text search. You can customize the hyperparameters as needed or write your own custom reranker. Here's how you can use any of the available rerankers:
```

Reply:
I feel like we're repeating ourselves a lot of times -- maybe it'll become clearer how to be less redundant once we push the docs to `main` and we can revisit the verbiage for that.